src/collection.ts

```diff
@@ -592,6 +593,27 @@ export class Collection<T extends TigrisCollectionType> implements ICollection {
 		});
 	}
 
+	/**
+	 * count the number of documents in a collection
+	 * @returns the number of documents in a collection
+	 */
+
+	count(filter: Filter<T>): Promise<number> {
```

Comment:
Filter could be optional, right?

```suggestion
	count(filter?: Filter<T>): Promise<number> {
	    if (!filter) {
		filter = { op: SelectorFilterOperator.NONE };
	    }
```

Reply:
Oh YES! that skipped my mind.